ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/editor/js/editDialog.js

```diff
@@ -47,5 +50,24 @@
         Utils.handlePatternFormat(dialog,DATEPICKER_LANGDISPLAYVALUE,DATEPICKER_LANG);
     }
 
-    Utils.initializeEditDialog(EDIT_DIALOG)(handleDisplayPatternDropDown,handleDisplayFormat,handleEditPatternDropDown,handleEditFormat,handleLang);
+    function handleDatePlaceholders(dialog){
+        var defaultDateInput = dialog.find(DATEPICKER_DEFAULTDATE + " input")[0],
+            minDateInput = dialog.find(DATEPICKER_MINDATE + " coral-datepicker")[0],
+            maxDateInput = dialog.find(DATEPICKER_MAXDATE + " coral-datepicker")[0],
+            defaultDateTooltip = dialog.find(DATEPICKER_DEFAULTDATE + " coral-tooltip")[0],
+            minDateTooltip = dialog.find(DATEPICKER_MINDATE + " coral-tooltip")[0],
+            maxDateTooltip = dialog.find(DATEPICKER_MAXDATE + " coral-tooltip")[0],
+            emptyText = Granite.I18n.get('YYYY-MM-DD', null, 'Input Default Date format'),
+            fieldDescription = Granite.I18n.get('Please enter the date in the required format "yyyy-mm-dd".', null, 'Input Default Date format');
+
+        defaultDateInput.placeholder = emptyText;
+        defaultDateInput.setAttribute('aria-label', emptyText);
```

Comment:
aria label should be something localized value of something like:
"Set default date in " + emptyText + " format"

and similarly for minimum and maximum date

Reply:
seems like a miss..

Reply:
miss as in?

Reply:
seems like a miss in granite component implementation, but we will not be fixing that. So keep the implementation as is, just improve the translated strings